.gitlab/benchmarks.yml

```diff
@@ -65,92 +73,50 @@ only-tracing:
   extends: .benchmarks
   variables:
     DD_BENCHMARKS_CONFIGURATION: only-tracing
+    DD_TRACE_ENABLED: "true"
 
 only-profiling:
   extends: .benchmarks
   variables:
     DD_BENCHMARKS_CONFIGURATION: only-profiling
     DD_PROFILING_ENABLED: "true"
```

Comment:
Originally it was

```
echo -e "\ngem 'dogstatsd-ruby'\ngem 'ddtrace', $TARGET_VERSION\n" >> /opt/gitlab/embedded/service/gitlab-rails/Gemfile
```

without auto_instrument.

Reply:
Profiling changes were discussed in slack thread.